lib/raven/configuration.rb

```diff
@@ -137,7 +140,7 @@ def initialize
       self.processors = DEFAULT_PROCESSORS.dup
       self.ssl_verification = true
       self.encoding = 'gzip'
-      self.timeout = 1
+      self.timeout = 2
```

Comment:
im a bit concerned I had to do this, but we might update the default to 3 or something